### DISCUSSION TITLE
I'm keep getting rejected because the language of my GitHub name and the language of my student id card's name doesn't match

### DISCUSSION THREAD

**Discussion by dn1t** [community/community#145910] [GitHub Education]
### Hi everyone,

Hi, I'm Korean and am trying to renew my student developer pack with my school id card. But I'm keep getting rejected and the team says that it is because the name on my student id card and the name of my account's billing information doesn't match.

Because I got rejected even after setting my GitHub profile and billing information name to match my English name on my passport, I tried to change my GitHub name to Korean, but since my last name is just a single letter(just like every other Koreans), it says the last name is too short and I can't update to it.

Is there a way to resolve this problem?

**Comment by hetaoawa** [comment]
Try another spell type of your name
eg. Your name : ABC
You can try first name : AB  last name : C
or
first name : ABC last name : ABC

**Reply by dn1t** [reply]
> Try another spell type of your name
> 
> eg. Your name : ABC
> 
> You can try first name : AB  last name : C
> 
> or
> 
> first name : ABC last name : ABC

tried it but still got rejected

**Comment by VernSG** [comment]
try open github support ticket for this issue
